src/test/data/draft/responseDraft.ts

```diff
@@ -13,15 +16,45 @@ export const responseDraft = {
   moreTimeNeeded: {
     option: 'no'
   },
+  freeMediation: {
+    option: 'no'
+  }
+}
+
+const baseDefenceDraft = {
   response: {
     type: {
-      value: 'DEFENCE'
+      value: ResponseType.DEFENCE.value
     }
   },
   defence: {
     text: 'My defence'
+  }
+}
+
+export const defenceWithDisputeDraft = {
+  ...baseResponseDraft,
+  ...baseDefenceDraft,
+  rejectAllOfClaim: {
+    option: RejectAllOfClaimOption.DISPUTE
+  }
+}
+
+export const defenceWithAmountClaimedAlreadyPaidDraft = {
+  ...baseResponseDraft,
+  ...baseDefenceDraft,
+  rejectAllOfClaim: {
+    option: RejectAllOfClaimOption.ALREADY_PAID
   },
-  freeMediation: {
-    option: 'no'
+  howMuchPaidClaimant: {
+    option: HowMuchPaidClaimantOption.AMOUNT_CLAIMED
```

Comment:
That property is not converted because we handle only one `ALREADY_PAID` scenario online at the moment. I thought we will add that later if needed.